lib/Myriad/RPC/Implementation/Redis.pm

```diff
@@ -11,15 +11,103 @@ use Object::Pad;
 
 class Myriad::RPC::Implementation::Redis extends Myriad::Notifier;
 
+use experimental qw(signatures);
+
 use Future::AsyncAwait;
 use Syntax::Keyword::Try;
-
-use Database::Async;
-use Database::Async::Engine::PostgreSQL;
+use Role::Tiny::With;
 
 use Log::Any qw($log);
+use Sys::Hostname;
+
+with 'Myriad::RPC';
+
+has $redis;
+has $group_name;
+has $whoami;
+has $service;
+has $rpc_map;
+method rpc_map :lvalue { $rpc_map }
+
+method BUILD(%args) {
+    $whoami = hostname;
+    $group_name = 'processors';
```

Comment:
This can be avoided by doing simply:

```perl
has $whoami = hostname();
has $group_name = 'processors';
```

then no need for the `BUILD` at all